card/src/main/java/com/adyen/checkout/card/internal/ui/DefaultCardDelegate.kt

```diff
@@ -547,8 +552,10 @@ internal class DefaultCardDelegate(
         )
     }
 
-    private fun isCvcHidden(): Boolean {
-        return componentParams.isHideCvc
+    private fun isCvcHidden(cvcUIState: InputFieldUIState = outputData.cvcUIState): Boolean {
+        val hiddenAfterBinLookup =
```

Comment:
Do we need these checks here or could `cvcUIState == InputFieldUIState.HIDDEN` be enough?